internal/controller/dashboard_controller.go

```diff
@@ -0,0 +1,344 @@
+/*
+Copyright 2024.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
There's no need for this, we already have the license at the repository root

Reply:
This is is all controllers, so I'd rather remove them in a separate PR.

Reply:
=> https://github.com/giantswarm/observability-operator/pull/213

Reply:
Still it should be removed here also